hr_contract_cae/models/hr_contract.py

```diff
@@ -1,11 +1,82 @@
 from dateutil.relativedelta import relativedelta
 
-from odoo import api, fields, models
+from odoo import api, fields, models, _
+from odoo.exceptions import ValidationError
+
+
+class ContractType(models.Model):
+
+    _inherit = "hr.contract.type"
+
+    echelon = fields.Selection(
+        [("main", "Main"), ("amendment", "Amendment")],
+        string="Payment Mode",
+        default="main",
+    )
+    max_usage = fields.Integer(
+        string="Maximum Usage", default=1, required=True
```

Comment:
I'd default to False, and make it not required. We don't have a limit on many contract types.